.circleci/config.yml

```diff
@@ -140,6 +136,59 @@ jobs:
       - store_test_results:
           path: /tmp/logs
 
+  gcp_test:
+    <<: *env
+    docker:
+      - image: cimg/base:2020.01
+
+    steps:
+      - attach_workspace:
+          at: /home/circleci
+
+      # install the google cloud sdk
+      - run:
+          command: |
+            echo 'deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main' | sudo tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
+            sudo apt-get install -y apt-transport-https ca-certificates gnupg
+            curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key --keyring /usr/share/keyrings/cloud.google.gpg add -
+            sudo apt-get update && sudo apt-get install -y google-cloud-sdk
```

Comment:
I'm wondering if I could add a new helper to `module-ci` and reference it here to install the Google Cloud SDK.

Reply:
Is this something we need in every GCP repo? If so, adding it to `module-ci` makes sense.

Reply:
@brikis98 yes, we'll ultimately adopt whatever logic I figure out here for all the GCP repos. I'll move it to `module-ci` then.

Reply:
Proposing a `gcp-helper` module in: https://github.com/gruntwork-io/terraform-aws-ci/pull/264